addon/models/property.js

```diff
@@ -14,6 +14,14 @@ export default class Property {
     return this._property.type;
   }
 
+  get title() {
+    return this._property.title;
+  }
+
+  get default() {
+    return this._property.default;
```

Comment:
:+1: - Need to add a test to the property tests (just needs to confirm that the getter works).